src/App.tsx

```diff
@@ -1,15 +1,47 @@
-import React from 'react';
-import './App.css';
+import React, {useEffect} from 'react';
+import {
+  RouterProvider,
+} from "react-router-dom";
+import {useAppDispatch} from "./Hooks/hook";
+import auth from "./Firebase";
+import {userAuth, userLogOut, userToken} from "./Redux/Store/auth/authSlice";
+import axios from "axios";
+import {appRouter} from "./Routes/Router";
+
+
+function App():JSX.Element {
+  const dispatch = useAppDispatch()
+  useEffect(()=> {
+    auth.onAuthStateChanged(authUser => {
+      if(authUser){
+        // console.log(authUser);
+        authUser.getIdTokenResult().then(result=> {
+          // console.log(result);
```

Comment:
same mistake that we have in BE repo, let's don't keep this kind of logs commented.

Reply:
All console and redundant comments are removed